addons/auth_signup_turnstile/static/src/signup.js

```diff
@@ -0,0 +1,94 @@
+/** @odoo-module **/
```

Comment:
This whole file is a big copy/paste of the file `addons/website_cf_turnstile/static/src/js/turnstile.js`

There is no way to avoid it ? It's a bit of a shame.

Reply:
> However is this what we want to do in stable?

As long as you do not break the API / imports to me it's fine. That being said I can indeed live with that copy/paste for stable if really it's too complicated to do otherwise. It's just that if it doesn't cost a lot then it's way better to avoid it.